e2eAIOK/common/default.conf

```diff
@@ -0,0 +1,74 @@
+## show all the configurations and default values in common lib
+## include required and optional settings
+
```

Comment:
all default config files of aiok are under e2eAIOK/conf folder, can we put this config file there?